infrastructure/azuredeploy.json

```diff
@@ -212,7 +212,7 @@
                     "properties": {
                         "WEBSITE_USE_ZIP": "[parameters('deploymentZipUrl')]",
                         "WEBSITE_NODE_DEFAULT_VERSION": "8.9.4",
-                        "FUNCTIONS_EXTENSION_VERSION": "~2",
+                        "FUNCTIONS_EXTENSION_VERSION": "~3",
```

Comment:
Yeah so we need to read the breaking change logs to check the impact of this.

Reply:
The breaking changes is [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions#migrating-from-2x-to-3x). I couldn't find any breaking changes in our code base but worth testing it in action.